setup.cfg

```diff
@@ -16,6 +16,7 @@ classifiers =
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
```

Comment:
Do we want to bump `python_requires`? Or let Python 3.9 continue on a best-effort basis?

Reply:
I think it's fine to continue 3.9.

Reply:
In that case shall we at least leave a pylint (and possibly a pytest) job lying around for Python 3.9

Reply:
> In that case shall we at least leave a pylint (and possibly a pytest) job lying around for Python 3.9

Just added them.